webcompat/form.py

```diff
@@ -25,7 +25,11 @@
 AUTH_REPORT = 'github-auth-report'
 PROXY_REPORT = 'github-proxy-report'
 SCHEMES = ('http://', 'https://')
+<<<<<<< HEAD
```

Comment:
Yeah, it's creating conflicts as it replays the patch history for some reason. Not a huge issue, I can clean it up.